src/pykeen/nn/__init__.py

```diff
@@ -50,6 +55,9 @@
     "RGCNRepresentation",
     "SubsetRepresentation",
     "TokenizationRepresentation",
+    "UniRelationalMessagePassingRepresentation",
```

Comment:
I would rename these to `SimpleMessagePassingRepresentation`, `TypedMessagePassingRepresentation`, and `FeaturizedMessagePassingRepresentation`

Reply:
cf. 779df2f